### DISCUSSION TITLE
类似于yt-dlp的配置文件？

### DISCUSSION THREAD

**Discussion by bottomash** [yutto-dev/yutto#36] [Ideas]
现在可配置的选项已经有很多了，比如我自己不喜欢弹幕就可以用 `--no-danmaku` 参数不下载弹幕、我想要下载到指定的文件夹可以用 `-d` 参数下载到指定文件夹，还可以引入 `-c `参数设置cookie等等。
但是现在好像没有yt-dlp那样可以把这些参数写入到一个文件具体的文件的设置，更改默认的下载行为。可以增加这样的配置文件设置吗？这样可以简化下载流程。

**Comment by SigureMo** [comment]
emmm，其实近期确实有这样的想法，但是优先级很低，因为增加配置文件可能会稍微麻烦些……

你可以参考下我目前使用的一个非常简单的方案，这里是 bilili 版本的：[定义命令别名](https://bilili.nyakku.moe/guide/knack.html#%E5%AE%9A%E4%B9%89%E5%91%BD%E4%BB%A4%E5%88%AB%E5%90%8D)。这样可以很轻松的解决每次都要输入一些固定参数的问题。macOS 和 Linux 下直接按照这个做应该就可以了，Windows 可能需要自己找下配置别名的方式。

**Reply by bottomash** [reply]
好的，感谢回复，就先用别名解决吧。
